extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboard.ts

```diff
@@ -65,7 +67,24 @@ export class BdcDashboard {
 
 			refreshButton.onDidClick(() => this.model.refresh());
 
-			const toolbarContainer = modelView.modelBuilder.toolbarContainer().withToolbarItems([{ component: refreshButton }]).component();
+			const openTroubleshootNotebook = modelView.modelBuilder.button()
+				.withProperties({
+					label: localize('bdc.dashboard.troubleshootButton', "Troubleshoot"),
+					iconPath: IconPath.notebook,
+					height: '50'
```

Comment:
I assume you can just pass in a number

Reply:
You can but I like to be explicit with the units (I just missed putting px there)